signing_service/signing_service/signing_service.py

```diff
@@ -578,4 +597,9 @@ def _parse_arguments() -> argparse.Namespace:
         args.signing_service_private_key,
         args.ethereum_private_key,
         args.max_reconnect_attempts,
+        Notifier(
```

Comment:
We need two `Notifier` classes:
1. `EmailNotifier` - which instance will be created when parser finds `smtp-notifier` option
2. `ConsoleNotifier` - which will be created when parser finds `console-notifier` option
3. `smtp-notifier` and `console-notifier` options must be mutually exclusive

`Notifier` is just a simple interface that provides a method `send(message)`. The above mentioned implementations do it like that:
1. `ConsoleNotifier` "sends" the message to stdout via logger (simply - logs the message)
2. `EmailNotifier` sends the message through gmail account.

Reply:
Done.
But like I wrote above, `console-notifier` is gone since it has no argument and is our default if `smtp-notifier` is not listed. It's impossible to make `subparsers` also a `add_mutually_exclusive_group`. They suppose to be like that by default but all `subparsers` need at least one argument.